packages/components/src/components/navigation-item/navigation-item.test.ts

```diff
@@ -295,4 +296,29 @@ describe('<sd-navigation-item>', () => {
       });
     });
   });
+
+  // Test separate variant
+  describe('when given a value for the "children" slot, an "href" property and "separate" property is true', () => {
+    it('renders a div with slot closed and a link in a separate div', async () => {
+      const el = await fixture<SdNavigationItem>(
+        html`<sd-navigation-item href="#" separate>${defaultSlot}${childrenSlot}</sd-navigation-item>`
```

Comment:
Here is a typo, it should be separated. This worries me, in case the tests are still green.

Reply:
We currently have an issue in our CI pipeline with our tests failing to connect to our Vite instance. A short debugging didn't have great outcome. As we already planned to rewrite the architecture, we rely on doing local tests before merging. As our current focus is documentation, this feels reasonable.